### PR TITLE
Added Support for Atlassian Jira in V2

### PR DESCRIPTION
**Pull request by maira-samtek**

### CODE REVIEW

File: src/libs/jira-lib.ts [hunk 1]
```diff
@@ -248,18 +253,29 @@ export class Jira {
   async removeCurrentUserAsWatcher(issueId: string) {
     try {
       const currentUser = await this.getCurrentUser()
-      console.log(`Remove watcher ${currentUser.name} from ${issueId}`)
+      console.log(currentUser)
```
Comment (yonassrobi) [inline src/libs/jira-lib.ts]: @maira-samtek - is this console.log needed or was it left accidentally after debugging?

**Review by yonassrobi — COMMENTED** [review]
Left one comment, the rest looks good to me.

File: src/libs/jira-lib.ts (hunk 1 above)
Comment (maira-samtek) [reply src/libs/jira-lib.ts]: Removed the log, this was to observe fields for different jira accounts

**Review by yonassrobi — APPROVED** [review]
LGTM

**Pull request merged**